module-2/backend/week_one.md

```diff
@@ -1,34 +1,111 @@
 ## Week One - Module 2 Recap
 
-Fork this respository. Answer the questions to the best of your ability. Try to answer them with limited amount of external research. These questions cover the majority of what we've learned this week (which is a TON!). 
+Fork this respository. Answer the questions to the best of your ability. Try to answer them with limited amount of external research. These questions cover the majority of what we've learned this week (which is a TON!).
 
-Note: When you're done, submit a PR. 
+Note: When you're done, submit a PR.
 
 1. List the five common HTTP verbs and what the purpose is of each verb.
+
+  get - This is used to read a record
+  post - Used to create a record
+  put - Used to update a record
+  patch - Also used to update a record??
+  delete - Used to delete a record
+
 2. What is Sinatra?
+
+  Sinatra is a DSL that allows us to give Ruby apps a web interface in a lightweight way.
+
 4. What is MVC?
+
+  This is the model-view-controller structure that we use in Sinatra (and maybe Rails?) that is a sort of a way to maintain the spirit of single responsibility in our apps. We break things into a controller, model(s), and views.
+
 5. Why do we follow conventions when creating our actions/path names in our Sinatra routes?
+
+  I would imagine it is to make our lives (as programmers) easier and make it less likely for bugs and other errors to cripple apps.
+
 6. What types of variables are accessible in our view templates without explicitly passing them?
+
+  anything in the params.

```

Comment:
Those are accessible in our controller, but *instance* variables are available in our views.